docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -0,0 +1,70 @@
+# Archiveren (voor dummies)
+
+## Gerelateerde attributen
+
+* `Zaak.Archiefnominatie` Aanduiding of het zaakdossier blijvend bewaard of na een bepaalde termijn vernietigd moet worden:
+
+    * `null`
+    * `Blijvend bewaren`
+    * `Vernietigen`
+    
+* `Zaak.Archiefstatus` De fase waarin het zaakdossier zich qua archivering bevindt:
+
+    * `nog te archiveren` (standaardwaarde)
+    * `gearchiveerd`
+    * `gearchiveerd (procestermijn onbekend)`
+    * `vernietigd`
+    * `overgedragen`
+    
+  Als de `Archiefstatus` `gearchiveerd` of `gearchiveerd (procestermijn onbekend)` is, dan is het zaakdossier niet-wijzigbaar.
```

Comment:
Dit kan volgens mij niet kloppen, heb je hierbij een referentie? In de RGBZ-documentatie lees ik het volgende:
> De vernietigingsdatum (attribuutsoort 'Archiefactiedatum') is mede afhankelijk van de datum waarop het procesobject (waarden van 'Procesobjectaard') vervalt (einde Selectielijst-procestermijn). In sommige gevallen is die datum bij afronding van de zaak nog niet te bepalen. 'Archiefstatus' krijgt dan de waarde "gearchiveerd (procestermijn onbekend)". Zo gauw de vervaldatum van het procesobject bekend is, kan de vernietigingsdatum bepaald worden en krijgt 'Archiefstatus' de waarde "gearchiveerd".
De eerder onbekende vervaldatum van een procesobject kan toch alleen bekend worden door een wijziging in de zaak? 

Het verschil tussen `gearchiveerd` en `nog te archiveren` heeft hier volgens mij te maken met eisen aan duurzame bewaarbaarheid van dossier bij de eerste groep, niet met eisen ten aanzien van onveranderlijkheid. Maar ik kan het mis hebben, dit is ver buiten mijn comfort zone.

Reply:
Ik baseer deze zin op RGBZ 2.0.2 deel II, pagina 576:

> Archiefstatus
> 
> Naam | Definitie
> -- | --
> nog   te archiveren | De   zaak cq. het zaakdossier is nog niet als geheel gearchiveerd.
> gearchiveerd | De   zaak cq. het zaakdossier is als geheel niet-wijzigbaar bewaarbaar gemaakt.
> gearchiveerd   (procestermijn onbekend) | De   zaak cq. het zaakdossier is als geheel niet-wijzigbaar bewaarbaar gemaakt   maar de vernietigingsdatum kan nog niet bepaald worden.
> vernietigd | De   zaak cq. het zaakdossier is vernietigd.
> overgedragen | De   zaak cq. het zaakdossier is overgebracht naar een archiefbewaarplaats.

Ik heb de omschrijving van de 2 "gearchiveerd" enumeraties samengevat.

Reply:
@hdksi Is het issue hiermee resolved of heb je nog aanvullingen?